lib/resources/r4/financial_transaction.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: FinancialTransaction
+field_name_base_url: http://hl7.org/fhir/R4/basic.html
+fields:
+- name: financial-transaction-method
+  description: Describes the method of payment for the financial transaction.
+  url: https://fhir-ehr.cerner.com/r4/StructureDefinition/financial-transaction-method
```

Comment:
Can we get these urls updated in accordance with Account?

Reply:
https://github.com/cerner/fhir.cerner.com/pull/297/commits/4adf16f12daf8a74d8b1caef1c28f22dd3d0b8fd#diff-5125c86cde87f8439cb9d1bb859194e9R7

Reply:
+1